src/main/java/control/Control.java

```diff
@@ -0,0 +1,38 @@
+package control;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Scanner;
+
+public class Control {
+    private static String printAnswer(double firstNumber, String operationSign, double secondNumber, double answer) {
+        String content = firstNumber + " " + operationSign + " " + secondNumber + " = " + answer;
+        System.out.println(content);
+        return content;
+    }
+
+    private static void writeToFile(Path path, String content) throws IOException {
+        Files.writeString(path, content + "\n", StandardOpenOption.APPEND, StandardOpenOption.CREATE);
+    }
+
+    public static void readFile(Scanner scanner, Path path, int lines) throws IOException {
```

Comment:
Scannera używasz tylko w tej metodzie, więc niepotrzebnie jest to parametr, mogłaby to być zmienna lokalna.
Parametr lines nie jest do niczego wykorzystywany. Inkrementujesz tę wartość, ale nigdzie jej nie używasz.